packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md

```diff
@@ -45,9 +49,21 @@ The third generation of the Sign-In Widget offers the same user experience as th
 
 See [Sign-In Widget, third generation](https://help.okta.com/okta_help.htm?type=oie&id=ext-compare-siw) to verify if the third generation is right for your org.
 
+### Palette generation and accessibility
+
+The third generation of the Sign-In Widget offers autogenerated palettes that preserve accessible color contrast ratios. See the [Odyssey by Okta design system](https://odyssey.okta.design/latest/foundations/accessibility/color-9O9a4XAU).
+
+Consider the following when adjusting the theming of your org:
```

Comment:
```suggestion
Consider the following when adjusting the theme of your org:
```

Reply:
I'm recommending this change because on the product side we talk about setting the **theme**, which includes the colors, logos, favicons, and background. However, you may be referring to the act of theming in a more general sense. If that's the case, okay to leave as is.